### PR TITLE
Always use rgb values for colors 

### PR DESCRIPTION
**Pull request by juliexxia**
Greens weren't matching up before. I also am not clear how createCheckboxForUserFeatures was working before, so updated. 

Now that layer green is very similar to the green of user features, make user features a nice cyan ('#00FFFF')

image

### CODE REVIEW

File: docs/firebase_layers.js [hunk 1]
```diff
@@ -123,21 +124,36 @@ function getLinearGradient(colorFunction) {
...
 }
+
+/**
+ * Converts a color to a css-recognizable rgb string.
+ * @param {string} color
+ * @return {string}
+ */
+function colorToRgbString(color) {
+  const rgbArray = colorMap.get(color);
+  if (!rgbArray) {
```
Comment (janakdr) [inline docs/firebase_layers.js]: this is an escape hatch because a lot of consumers can handle a color string?

File: docs/run.js [hunk 2]
```diff
@@ -272,7 +276,7 @@ function addLayers(map, firebaseLayers) {
     }
     createNewCheckboxForLayer(properties, sidebarDiv, map);
   }
-  createCheckboxForUserFeatures(sidebarDiv);
+  createCheckboxForUserFeatures(sidebarDiv, map);
```
Comment (janakdr) [inline docs/run.js]: why this change? I don't think it takes the map as an argument?

File: docs/firebase_layers.js (hunk 1 above)
Comment (janakdr) [reply docs/firebase_layers.js]: Could you instead/in addition add documentation to the `@param` tag for color that says what it's expected to be?

Comment (juliexxia) [reply docs/firebase_layers.js]: Done

File: docs/firebase_layers.js [hunk 3]
```diff
@@ -123,21 +124,37 @@ function getLinearGradient(colorFunction) {
...
+      gradientString += ', ' + rgb + ', ' + rgb;
       break;
   }
   return gradientString + ')';
 }
+
+/**
+ * Converts a color to a css-recognizable rgb string.
+ * @param {string} color one of the keys of our colorMap or a hex string
+ * for special cases like the user features color or the score color.
```
Comment (janakdr) [inline docs/firebase_layers.js]: nit: I think this should be indented 4 spaces.
